### PR TITLE
Set up

### PR DESCRIPTION
**Pull request by Akorede3133**
In  this PR , the app was bootstrapped  using create-react-app

### CODE REVIEW

**Review by Strangeal — COMMENTED** [review]
Hi @Akorede3133, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] **It appears that your submitted project is missing this essential component. To rectify this, kindly consider the following improvements**:
   - Utilize the [README](https://github.com/microverseinc/readme-template) template provided in the project requirements, This will ensure consistency and completeness in your documentation. 
 Customize the template to suit your project details. Provide a clear and concise overview of your project, including its description, key features, future features, and any relevant instructions or prerequisites for users. 
   - Consider consuming sections such as installation instructions, usage examples, configuration details, and information about any external dependencies or libraries used. Include information about the project's license, copyright, or any other legal considerations. 
   - Proofread and format your README to ensure it is easy to read, well-organized, and free of errors. Remember, documenting your project through a professional README is crucial for other developers to understand, use, and contribute to your project effectively.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,71 +1,182 @@
...
+I would like to thank Microverse for this great opportunity!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Akorede3133/math-magicians/blob/main/LICENSE) licensed.
+
```
Comment (mikethreels) [inline README.md]: - [ ] The link to your MIT license is not working as expected, please make sure the functionality is the same as in the template provided by Microverse.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Akorede , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good job setting up the project.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — APPROVED** [review]
Hi @Akorede3133 👋🏼👋🏼👋🏼, 

approved


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅✅

- [x] mit license link has been fixed👍🏼.
- [x] Linters are passing👍🏼.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**